### DISCUSSION TITLE
about vue-i18n-routing

### DISCUSSION THREAD

**Discussion by kazupon** [intlify/routing#1] [Show and tell]
Hi, everyone!

I’m working on `vue-i18n-rouging` as an `intlify/routing` project.

I noticed that nuxt/i18n routing is generics for vue community when I was prototyping `nuxt/i18n` for nuxt bridge / nuxt3.

so, for this reason, I’m making a i18n routing library.

Currently, Unfortunately, I can not prepare the docs 😅 , but there are the [example apps](https://github.com/intlify/routing/tree/main/playground) for Vue 2 and Vue 3 ✨ 
if you are interested in it, welcome feedback.

vue-i18n-routing is going to be used for [nuxt/i18n](https://github.com/nuxt-community/i18n-module)

**Comment by rchl** [comment]
Thanks for the heads up.

I've looked a bit at the code but it's a lot and I'm not sure I'll be able to dig deeper into it and review it properly. It looks promising though. Good work.

**Reply by kazupon** [reply]
@rchl 
Thanks!

I have just integrated and pushed this package to the `next` branch of nuxt/i18n.
https://github.com/nuxt-community/i18n-module/tree/next

TODO list here:
https://github.com/nuxt-community/i18n-module/blob/next/TODO.md

About the next, I'll discuss at the discussions
https://github.com/nuxt-community/i18n-module/discussions/1287
